app/__init__.py

```diff
@@ -705,6 +733,54 @@ def create_presigned_post():
     # The response contains the presigned URL and required fields
     return response
 
+
+def create_presigned_url_local(filename, expiration=3600):
+    """Generate a presigned URL to share an S3 object
+
+    :param expiration: Time in seconds for the presigned URL to remain valid
+    :return: Presigned URL as string. If error, returns None.
+    """
+    bucket_name = "snorkel"
+    object_name = filename
+    # Generate a presigned URL for the S3 object
+    s3_client = boto3.client('s3',)
+    try:
+        response = s3_client.generate_presigned_url('get_object',
+                                                    Params={'Bucket': bucket_name,
+                                                            'Key': object_name},
+                                                    ExpiresIn=expiration)
+    except ClientError as e:
+        logging.error(e)
+        return None
+
+    # The response contains the presigned URL
+    return {'data': response}
+
+
+
+@app.route("/s3-download")
```

Comment:
If this isn't really used rn, maybe let's leave a comment indicating that it's for testing